docs/releases/release-1.0.md

```diff
@@ -0,0 +1,50 @@
+# Kubeflow Pipelines 1.0 Release Notes
+
+## Versioning policy
+
+Kubeflow Pipelines 1.0 is the first major release of Kubeflow Pipelines. It follows the Kubeflow Pipelines [versioning policy](https://github.com/kubeflow/pipelines/blob/master/docs/releases/versioning-policy.md).
+
+## Deployment options
+
+There are two available ways to [deployment Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/installation/overview/).
+
+* [Standalone Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/installation/standalone-deployment/), which allows users the maximum flexibility in configuring the Kubernetes clusters and a Kubeflow Pipelines instance.
+
+* [AI Platform Pipelines](https://cloud.google.com/ai-platform/pipelines/docs), which minimizes the user effort in deploying a Kubeflow pipelines instance.
+
+## Core features and APIs
+
+Core features and APIs are the fundamental functionalities of Kubeflow Pipelines. The most fundamental functionality is to manage the Kubeflow Pipelines resources (that is, pipeline, pipeline version, run, job, experiment) via the operations listed below. Active supports are available for core features and APIs.
+
+|                  | Upload | Create | Get | List | Delete | Archive | Unarchive | Enable | Disable | Terminate | Retry |
+|:----------------:|:------:|:------:|:---:|:----:|:------:|:-------:|:---------:|:------:|:-------:|:---------:|:-----:|
+| Pipeline         | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |
+| Pipeline Version | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |
+| Run              | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |        |         | - [x]     | - [x] |
+| Job              | - [x]  |  - [x] |- [x]|- [x] | - [x]  |         |           | - [x]  | - [x]   |
+| Experiment       | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |
+
+* Refer to the Kubeflow Pipelines [introduction]((https://www.kubeflow.org/docs/pipelines/overview/)) for
+  - An [overview](https://www.kubeflow.org/docs/pipelines/overview/pipelines-overview/) and [main concepts](https://www.kubeflow.org/docs/pipelines/overview/concepts/), for example, what is a pipeline or run in Kubeflow Pipelines.
+  - How to manage the Kubeflow Pipelines resources via the the [Kubeflow Pipelines UI](https://www.kubeflow.org/docs/pipelines/overview/interfaces/).
+* Refer to the Kubeflow Pipelines [SDK samples and tutorials](https://www.kubeflow.org/docs/pipelines/tutorials/sdk-examples/)
+for how to manage the Kubeflow Pipelines resources via the Kubeflow Pipelines client.
+* A complete API method list and a detailed description of parameters to those API methods are available at the Kubeflow Pipelines [API reference](https://www.kubeflow.org/docs/pipelines/reference/api/kubeflow-pipeline-api-spec/).
+
+## Experimental features
+
+Experimental features are the feature in the early phases of development and may change drastically in the future.
```

Comment:
I think we need alpha and beta for finer introduction, because these areas have very different maturity.

job, built-in visualization, sdk, upgrade support should be BETA

multi user, custom python visualization, caching, ml metadata and lineage should be ALPHA

Any concerns or disagreements?